app/Http/Controllers/AboutController.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * AboutController.php
+ *
+ * -Description-
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2018 Tony Murray
+ * @author     Tony Murray <murraytony@gmail.com>
+ */
+
+namespace App\Http\Controllers;
+
+use App\Models\Application;
+use App\Models\Callback;
+use App\Models\Device;
+use App\Models\DiskIo;
+use App\Models\EntPhysical;
+use App\Models\Eventlog;
+use App\Models\HrDevice;
+use App\Models\Ipv4Address;
+use App\Models\Ipv4Network;
+use App\Models\Ipv6Address;
+use App\Models\Ipv6Network;
+use App\Models\Mempool;
+use App\Models\Port;
+use App\Models\Processor;
+use App\Models\Pseudowire;
+use App\Models\Sensor;
+use App\Models\Service;
+use App\Models\Storage;
+use App\Models\Syslog;
+use App\Models\Toner;
+use App\Models\Vlan;
+use App\Models\Vrf;
+use App\Models\WirelessSensor;
+use DB;
+use LibreNMS\Config;
+use LibreNMS\Util\Version;
+
+
+class AboutController extends Controller
+{
+
+    public function index()
+    {
+        $callback_uuid = null;
+
+        $callback_status = Callback::where('name', 'enabled')->first()->value;
+        if($callback_status && $callback_status != 2) {
+            $callback_uuid = Callback::where('name', 'uuid')->first();
+        }
+
+        $version = Version::get();
+
+        return view('about.index', [
+            'callback_status' => $callback_status,
+            'callback_uuid'   => $callback_uuid,
+
+            'db_schema' => vsprintf('%s (%s)', $version->database()),
+            'git_log'   => $version->gitChangelog(),
+            'git_date'  => $version->gitDate(),
+            'project_name' => Config::get('project_name'),
+
+            'version_local'     => $version->local(),
+            'version_mysql'     => DB::select('select version()')[0]->{'version()'},
```

Comment:
Might through error if no response.
This might be better `current(DB::selectOne('select version()'))`

Reply:
Yes, way better.